### ISSUE TITLE
RedisBroadcaster connect always to localhost

### ISSUE THREAD

**Issue by christianHennig** [Atmosphere/atmosphere-extensions#192]
I use the RedisBroadCaster:
getBroadcasterConfig().addFilter( new RedisFilter( URI.create( "http://x.x.x.x:6379" ) ) );

The Ip will ignored and overwriten with localhost:
https://github.com/Atmosphere/atmosphere-extensions/blob/master/redis/modules/src/main/java/org/atmosphere/plugin/redis/RedisBroadcaster.java#L42

**Comment by jfarcand** [comment]
Good catch! Can you do me a favor and do a pull request?  I will cut an official release in return.

**Comment by christianHennig** [comment]
Good point. I will do a pull request the next days.

**Comment by yissachar** [comment]
Can't this be configured through the servlet config? See https://github.com/Atmosphere/atmosphere-extensions/blob/master/redis/modules/src/main/java/org/atmosphere/plugin/redis/RedisUtil.java#L69 

I am setting `org.atmosphere.plugin.redis.RedisBroadcaster.server` to `http://myredisserver:6379` and it seems to work for me.

Still it is weird to pass in a URI when that URI is either overridden to `localhost` or pulled from the servlet config, so the code can probably be cleaned up.

**Comment by Flo354** [comment]
Any news on this issue ? I will copy the source meanwhile.
